src/Behat/Behat/EventDispatcher/Cli/StopOnFailureController.php

```diff
@@ -81,12 +81,7 @@ public function configure(Command $command)
     }
 
     /**
-     * Executes controller.
-     *
-     * @param InputInterface  $input
-     * @param OutputInterface $output
-     *
-     * @return null|integer
+     * {@inheritdoc}
```

Comment:
Prefer to use the base annotation for all controller classes

Reply:
The inline tag `{@inheritdoc}` is about inheriting the parent description, which is useful only to merge it with more description. It has nothing to do with inheriting tags like `@param` (which are inherited automatically anyway in phpdoc tools).
You should either remove the phpdoc entirely or use the `@inheritdoc` tag as an explicit marker meaning "the phpdoc is inherited from the parent, I did not forgot to add it".
My preference is to remove it entirely.

Reply:
OK, understood, I also prefer to remove it